cypress/support/commands.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference types="cypress" />
+
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace Cypress {
+  interface Chainable {
+    /**
+     * Custom command to login into Atlas with a test membership.
+     */
+    login(): Chainable<void>
+    /**
+     * Custom command to make the polkadot api available and add accounts, memberships, etc.. to it.
+     */
+    injectPolkadot(accounts: any[]): Chainable<void>
+  }
+}
+
+Cypress.Commands.add('injectPolkadot', (accounts = []) => {
+  const enable = async () => {
+    return {
+      accounts: {
+        get: () => new Promise((resolve) => resolve(accounts)),
+      },
+    }
+  }
+  cy.window().then((window) => {
+    window['injectedWeb3'] = {
+      'polkadot-js': {
+        enable,
+        version: '0.38.3',
+      },
+    }
+  })
+})
+
+Cypress.Commands.add('login', () => {
```

Comment:
I'm not sure about this, It seems non-semantic to me. Injecting Polkadot extension is not equal to log user in. WDYT?

Reply:
Yeah, I wasn't sure how to do it, as the injected polkadot gets the accounts on that enable fn, I don't know if it's possible to add them post-injection? that should way better

could possibly be something you could add to the signing transaction PR if you have some insight on it?

Reply:
Sure, let's do it this way